### DISCUSSION TITLE
can you please make a Mac version

### DISCUSSION THREAD

**Discussion by yamps-wastolen** [baffler/Transparent-Twitch-Chat-Overlay#296] [Ideas]
I feel bad since im probably the only Mac user but it would probably help a lot of people. it might be simple but I don't know how it works but I just want to be able to use it on Mac and most of the exe runners on Mac are outdated.

**Comment by IAmAProfessionalGamer** [comment]
You could do a dual boot of windows or an entire separate partition on a different HD/usbHD.

Wine is consistently updated and there are a few tools you can layer onto it to add GUI functions like WineZGui does. Bottles may also be an option, I don't have a Mac but I do have a Steamdeck. Both should be able to access the same repositories and run them bc at it's core Apple OS is essentially unix.

The main issue for the developer is that writing code for Apple requires the entire thing to be rewritten. I forget the name of the code but Apple uses it's own way of running the OS which requires a huge amount of work to recode from Windows to Mac.

**Reply by yamps-wastolen** [reply]
yeah I tried using wine and I for some reason didn't work though I do understand the effort and time put into making a Mac version sure would be nice to have an .exe to .app converter
